### ISSUE TITLE
Subscribers

### ISSUE THREAD

**Issue by NicholasLYang** [stuyspec/stuy-spec-api#99]
Um, yeah. Subscriptions don't actually do anything right now...We should probably change that. Also, instead of a subscribe button, we should turn that into a sign up button since subscriptions don't work. 

<img width="285" alt="screen shot 2018-02-26 at 7 34 21 pm" src="https://user-images.githubusercontent.com/7357863/36703953-132e3a08-1b2c-11e8-8b60-9bd7df459d5b.png">

**Comment by jkao1** [comment]
Subscriptions do create Subscribers. It's just that we have no newsletter to share.

**Comment by NicholasLYang** [comment]
Yeah that’s what I mean by “don’t do anything”. Maybe until newsletters are a thing we should hide the functionality

**Comment by jkao1** [comment]
The button is just nice for UI in my opinion. I fully acknowledge your point, but it's not cluttering the site; it's making the site less bare and more colorful.

**Comment by AndySok** [comment]
For devs: if you want to take this project, ask us to connect you with the EiCs so you understand what the subscribers are currently used for

**Comment by jsonkao** [comment]
Grandpa is proud

![selfie-0](https://i.imgur.com/hC4ebvo.jpg)
[_GitHub Selfies_](https://github.com/thieman/github-selfies/)

**Comment by AndySok** [comment]
oh we're just getting started >:)

**Comment by jsonkao** [comment]
Grandpa is still proud
